src/util/detect-bugs.js

```diff
@@ -1,23 +1,63 @@
-import TRANSITIONRUN_EVENT_LOOP_BUG from "./detect-bugs/transitionrun-loop.js";
-import UNREGISTERED_TRANSITION_BUG from "./detect-bugs/unregistered-transition.js";
+import detectTransitionRunLoopBug from "./detect-bugs/transitionrun-loop.js";
+import detectUnregisteredTransitionBug from "./detect-bugs/unregistered-transition.js";
 
-/**
- * Data structure for all detected bugs.
- * All bugs start off as true, and once their promises resolve, that is replaced with the actual value
- */
-export const bugs = {
-	TRANSITIONRUN_EVENT_LOOP: true,
-	UNREGISTERED_TRANSITION: true,
-};
+export default class Bugs {
```

Comment:
I could be missing something, but this class seems to be adding complexity without really providing much of a benefit, since the individual bugs are still ad-hoc properties on it. I could see the class being a better design if was some sort of generic bug detection class, though even then it seems overkill just for detecting two bugs.

Reply:
Oh yeah we don't actually need a class to have getters on an object, right? So you're right, a class is overkill. I will change it back to an object and use variables in the module scope instead of private properties.